From: Roland Illig <roland.illig@gmx.de>
Date: Sun, 17 May 2020 23:41:42 +0200
Subject: [PATCH] libetonyek: std::for_each

NUM3Parser.cpp: In member function 'virtual bool
libetonyek::NUM3Parser::parseDocument()':
NUM3Parser.cpp:46:3: error: 'for_each' is not a member of 'std'
   std::for_each(sheetListRefs.begin(), sheetListRefs.end(),
std::bind(&NUM3Parser::parseSheet, this, std::placeholders::_1));

std::for_each needs <algorithm>.

--- a/src/lib/NUM3Parser.cpp
+++ b/src/lib/NUM3Parser.cpp
@@ -7,6 +7,7 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/.
  */

+#include <algorithm>
 #include <functional>

 #include "NUM3Parser.h"
